client/src/partials/templates/bhLocationSelect.tmpl.html

```diff
@@ -3,11 +3,11 @@
   a child form is added for validation purposes, and to make the HTML easier
   to read.
 -->
-<fieldset ng-disabled="$ctrl.disable" ng-form="LocationSelectForm" data-bh-location-select>
+<fieldset ng-disabled="$ctrl.disable"  ng-form="{{$ctrl.name}}" ng-model="somevaluetwo" data-bh-location-select>
 
   <!-- allows a user to select a country -->
   <div class="form-group"
-    ng-class="{ 'has-error' : $ctrl.validationTrigger && LocationSelectForm.country.$invalid }"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && this[$ctrl.name].country.$invalid }"
```

Comment:
Nope, I have no experience using `ngInit`, just hoped it would work.  Darn, I wish you could dynamically assign directive priorities.

Okay, thanks for investigating this further.

Does the `$timeout()` call work in this case?  Do you prefer `this[$ctrl.name]` over `$timeout()`?